trento/xml/article_sap_trento.xml

```diff
@@ -96,7 +96,7 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
       </listitem>
       <listitem>
         <para>Configuration validation for &hana; Scale-Up
-          Performance-Optimized scenarios deployed on Azure, AWS, GCP or on-premise bare metal platforms, including KVM and Nutanix.
+          Performance-Optimized 2-node scenarios deployed on Azure, AWS, GCP or on-premise bare metal platforms, including KVM and Nutanix.
```

Comment:
Some points:

* Spelling: It's two-node, not 2-node. :wink: 
* The `on-premise` -> `on-premises`
* Performance-Optimized -> Performance-optimized
* What is a _two-node scenario_? Do you mean a scenario involving two-nodes? Or a two-node cluster? 

Especially the last point isn't clear to me. I can see two options:

```
Performance-optimized scenarios on two nodes deployed on...
```

or

```
Performance-optimized scenarios on two-nodes clusters deployed on...
```

Or we go into a completely different direction:

```
Optimizing performance on two-node clusters|setups.
```

Would that be sufficient, since public cloud and bare metal is covering all deployment scenarios?

What do you think is more appropriate?

Reply:
OK to points 1 and 3:

- Spelling: It's two-node, not 2-node. 😉
- Performance-Optimized -> Performance-optimized

Regarding point 2: I think "on-premise" is the right expression here.

Regarding point 4: I meant a two-node cluster. So let's make the following change:

Configuration validation for SAP HANA Scale-Up Performance-Optimized two-node clusters deployed on Azure, AWS, GCP or on-premise bare metal platforms, including KVM and Nutanix.

Reply:
Thanks, looks good! :+1: